base/event/FairMCEventHeader.cxx

```diff
@@ -57,7 +55,6 @@ FairMCEventHeader::FairMCEventHeader(Int_t iEvent,
                                      Int_t nPrim)
     : TNamed("MCEvent", "MC")
     , fRunId(0)
-    , fEventId(iEvent)
```

Comment:
I doubt you wanted to remove that one really?

I wonder, why this didn't throw a warning about unused parameter though…